draft-dekater-scion-controlplane.md

```diff
@@ -224,11 +222,17 @@ As SCION is an *inter-domain* network architecture, it only deals with *inter*-d
 
 SCION routers and endpoints connect to each other via links. A SCION path between two endpoints essentially traverses one or more links.
 
-In SCION, autonomous systems (ASes) are organized into logical groups called isolation domains or ISDs. Each ISD consists of ASes that span an area with a uniform trust environment (i.e., a common jurisdiction). An ISD is administered by a set of distinguished ASes called core ASes. Within and between ISDs, SCION supports three types of links: (1) core links, (2) parent-child links, and (3) peering links.
+In SCION, autonomous systems (ASes) are organized into logical groups called isolation domains or ISDs. Each ISD consists of ASes that span an area with a uniform trust environment (i.e., a common jurisdiction). An ISD is administered by a set of distinguished ASes called core ASes.
+
+SCION distinguishes three types of links between ASes: (1) core links, (2) parent-child links, and (3) peering links.
+
+- *Core* links connect two core ASes, which are either within the same or in a different ISD. Core links can exist for various underlying business relationships, including provider-customer (where the customer pays the provider for traffic) and peering relationships.
```

Comment:
"or in a different ISD" -> "or in different ISDs"

Reply:
Done.